server/utils/geocode.js

```diff
@@ -1,6 +1,6 @@
 const fetch = (...args) => import('node-fetch').then(({ default: fetch }) => fetch(...args));
 
-const cities = require('../../data/db.json');
+// const cities = require('../../data/db.json');
```

Comment:
I believe Stephan indicated that this line is no longer needed, so this can be removed if any comments need to be resolved, or in the next merge if not.